File: modules/publix/app/controllers/publix/Assets.java
package controllers.publix;

import javax.inject.Inject;

import play.api.mvc.*;

public class Assets {

	@Inject
	controllers.Assets assets;

	public Action<AnyContent> versioned(String path,
			controllers.Assets.Asset file) {
		return assets.versioned(path, file);
	}
	
	public Action<AnyContent> jatosjs(String path,
			controllers.Assets.Asset file) {
		return assets.versioned(path, file);
	}
}

File: modules/publix/app/controllers/publix/Assets.scala
package controllers.publix

import play.api.http.HttpErrorHandler

import javax.inject.Inject

// todo why do we need this?
class Assets @Inject()(errorHandler: HttpErrorHandler,
                       assetsMetadata: controllers.AssetsMetadata
                      ) extends controllers.AssetsBuilder(errorHandler, assetsMetadata)
